staging/src/k8s.io/cri-client/pkg/remote_runtime.go

```diff
@@ -136,6 +138,11 @@ func NewRemoteRuntimeService(endpoint string, connectionTimeout time.Duration, t
 	return service, nil
 }
 
+// Close will shutdown the internal gRPC client connection.
```

Comment:
do we need to ensure nothing is called after connection is closed?

Reply:
yes, this is the first thing I checked. I mostly worried about other clients safety